contact.html

```diff
@@ -65,26 +66,21 @@ <h1>Header</h1>
 		<br>
 		<!-- Submit Button -->
 		<div class="row">
+		<div class="submit">
 		<input type="submit" value="Submit">
 		</div>
+		</div>
 			
 		</form>
 				
 
 		<!-- Copyright Statement -->
+	
 	</div>
-		
-					<div class="column3">
-			<img src="images/placeholder.png">
-		</div>
-		
-		<div class="footer">
+		<div class="contact_footer">

```

Comment:
I didn't like the idea of having a seperate image when I had used a background image instead